book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Please, add `:rentals` for getter and setter. This variable could be writable too. 🐶

Reply:
:rentals were set as editable as well

Reply:
![setter](https://user-images.githubusercontent.com/57562869/136302829-c528f9b8-fff5-437a-a386-cf1747c95e52.png)

- This is the setter method for rentals